src/App.tsx

```diff
@@ -1,24 +1,208 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import {
+  getTodos, deleteTodo, patchTodo, addTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Filter } from './types/Filter';
+import { getVisibleTodos } from './utils/helpers';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
+import { ErrorType } from './types/ErrorType';
 
-const USER_ID = 0;
+export const USER_ID = 6344;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [numberOfActive, setNumberOfActive] = useState(0);
+  const [numberOfCompleted, setNumberOfCompleted] = useState(0);
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NONE);
+  const [hasError, setHasError] = useState(false);
+  const [filterValue, setFilterValue] = useState<Filter>(Filter.ALL);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [processingTodosIds, setProcessingTodosIds] = useState<number[]>([]);
+
+  const showError = useCallback((message: ErrorType) => {
+    setErrorType(message);
+    setHasError(true);
+    setTimeout(() => {
+      setHasError(false);
+    }, 3000);
+  }, []);
```

Comment:
if you get 2 errors within 3 seconds the second one will clear faster than expected

Reply:
🔥thank you